static/bidder-info/iqzone.yaml

```diff
@@ -13,4 +13,10 @@ capabilities:
     - banner
     - video
     - native
-
+userSync:
+  redirect:
+    url: "https://cs.iqzone.com/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redir={{.RedirectURL}}"
```

Comment:
it works:
![image](https://github.com/user-attachments/assets/19757863-b570-4ea2-a302-2be23bc364b4)